--- src/main/java/thaumcraft/common/lib/research/theorycraft/CardSynthesis.java
+++ src/main/java/thaumcraft/common/lib/research/theorycraft/CardSynthesis.java
@@ -89,8 +89,7 @@
             data.addInspiration(actResult.bonusInspiration);
         }

-        ItemStack res = ThaumcraftApiHelper.makeCrystal(aspect3);
-        givePlayerItem(player, res);
+        givePlayerItem(player);
         return true;
     }

@@ -98,8 +97,9 @@
         return player.getRandom().nextFloat();
     }

-    protected void givePlayerItem(Player player, ItemStack res) {
-        if (!player.getInventory().add(res)) {
+    protected void givePlayerItem(Player player) {
+        ItemStack res = ThaumcraftApiHelper.makeCrystal(aspect3);
+        if (player != null && !player.getInventory().add(res)) {
             player.drop(res, true);
         }
     }
 }
